test-apps/test-tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,20 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+#include "CANMsg.h"
+
+TutorialServo servo(PA_1);
+CAN can(PB_8, PB_9); 
+
+int main(){
+
+    CANMessage msg;    
```

Comment:
```suggestion
    CANMsg msg;    
```

Recommend using our custom `CANMsg` wrapper class instead of mbed's `CANMessage`. The `getPayload()` function is actually part of the `CANMsg` class.